presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
+  so INNER join should not have PARTITIONED distribution type".
+* Add support for recoverable grouped execution. This can be enabled with the
```

Comment:
@shixuan-fan - can you verify if this point and the one below accurately represent your message.

Reply:
Could we have a clearer explanation of what recoverable grouped execution means.  e.g. "automatic retries for failed buckets when using grouped execution"

Reply:
The one for `grouped_execution_for_eligible_table_scans` looks good. 

For recoverable grouped execution one, this could work but I'm not sure if we also need to specify the limitation. Maybe we could add `Currently this is only available  for limited query shapes when input and output connectors are Hive connector`. This is still not 100% accurate/clear. The limitation is specified in comment in `PlanFragmenter` and has multiple bullet points, but I think it is not suitable for broad audience of release note. (https://github.com/prestodb/presto/blob/1e3f420e67b5de8fae9b9f407e7f80d94ed508d5/presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenter.java#L216-L225).

I also want to see what the committers think about how we should clarify this without exposing too many unnecessary details.